### PR TITLE
feat(jest): 100% coverage for Iridium/chat/types

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

Self-explanatory

**Which issue(s) this PR fixes** 🔨

AP-2132

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by stavares843** [comment]
this needs to be added in iridium repo instead, please add this on that repo

File: libraries/Iridium/chat/types.test.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { ChatError } from './types'
+
+describe('', () => {
```
Comment (stavares843) [inline libraries/Iridium/chat/types.test.ts:3]: can we add a name on the describe instead of being empty?

**Review by stavares843 — CHANGES_REQUESTED** [review]

**Comment by drepram** [comment]
After I committed my changes, 1 test failed `libraries/BlockchainClient/adapters/Phantom/PhantomAdapter.test.ts`.  

I do not see the relation between the file in this PR with that test that's returning an error, but I find the use in bringing it up so that we may resolve it quickly.

**Comment by stavares843** [comment]
Captura de ecrã 2022-08-18, às 12 03 41

its all green on CI

is also all green locally, not sure why you got that error

**Review by stavares843 — APPROVED** [review]

**Pull request merged**